cli/tests/activate/nested-activate-fish.exp

```diff
@@ -0,0 +1,40 @@
+# Activate a project environment using --dir
+
+set dir [lindex $argv 0]
+set flox $env(FLOX_BIN)
+set timeout 10
+
+spawn $flox activate --dir $dir/outer
+expect_after {
+  timeout { exit 1 }
+  eof { exit 2 }
+  "*\n" { exp_continue }
+  "*\r" { exp_continue }
+}
+expect "You are now using the environment"
+
+send "$flox activate --dir $dir/inner\r"
+expect "You are now using the environment"
+
+send "inner\r"
+expect "the inner alias is defined"
+
+send "outer\r"
+expect "the outer alias is defined"
+
+send "current\r"
+expect "the current alias is inner"
+
+puts "INFO: exiting inner env"
+send "exit 0\r"
+
```

Comment:
question nonblocking: do you need an expect between these two sends?

Reply:
The only thing we can expect is a prompt which would differ by shell. I suppose we could expect a newline?

Reply:
I'll go ahead and resolve this conversation .. please reopen if you would want to add an `expect "\n"` (or equivalent ... would have to research whether to use "\r" or "\n").